docs-md/developer-guide/joins/partition-data.md

```diff
@@ -16,67 +16,70 @@ records based on the joining column. To ensure that records with the same
 join column are co-located on the same stream task, the join column must
 coincide with the column that the sources are partitioned by.
 
-Primary key
------------
+Keys
```

Comment:
Note: changed from _Primitive Keys_ to _Keys_ as a _primitive_ key means something very specific in SQL: its a _unique_ non-null key.  

In KSQL tables have primitive keys and streams do not. This is because stream are just a stream of facts/events. Each stands alone.  Keys are not _unique_: duplicate keys are allowed.

Hence, we should standardize on only using _primitive_ keys when talking about _tables_ and just _keys_ when talking about streams.

Reply:
I assume you meant `s/primitive/primary/g`?